test/utils.jl

```diff
@@ -0,0 +1,134 @@
+using Pkg, Test, LibGit2
+
+"""
+`copy_test_package` copied from Pkg.jl as cannot be imported during tests.
```

Comment:
why not?

Reply:
It is defined in `Pkg.jl/test/utils.jl` so to load it we'd need to `include` it (its not available as in import from `Pkg`). However I can't get the package directory of `Pkg` as `pkgdir(Pkg)` returns `nothing`. Unless there is another way of getting the path I don't think we can use it.

Reply:
Ah it is in the test dir

```suggestion
`copy_test_package` copied from [`Pkg.jl/test/utils.jl`](https://github.com/JuliaLang/Pkg.jl/blob/v1.4.2/test/utils.jl#L209).
```